addons/calendar/models/calendar_event.py

```diff
@@ -324,6 +324,27 @@ def _compute_attendee(self):
             attendee = mapped_attendees[meeting.id]
             meeting.attendee_status = attendee.state if attendee else 'needsAction'
 
+    def _search_attendee_status(self, operator, value):
+        negative = operator in expression.NEGATIVE_TERM_OPERATORS
+
+        # In case we have no value or non supported operator
+        if not value or operator not in ['=', '!=', 'in', 'not in']:
+            return expression.TRUE_DOMAIN if negative else expression.FALSE_DOMAIN
+
+        all_status = {'needsAction', 'tentative', 'declined', 'accepted'}
```

Comment:
You already have the status names defined in Attendee.STATE_SELECTION
```suggestion
        all_status = set([status[0] for status in Attendee.STATE_SELECTION])
```